core/includes/bootstrap.inc

```diff
@@ -3307,15 +3333,15 @@ function _backdrop_bootstrap_sanitize_request() {
 /**
  * Sanitizes unsafe keys from request input.
  *
- * @param mixed $input
+ * @param array $input
  *   The input data array from a request to be sanitized.
- * @param string[] $allowlist
+ * @param array $allowlist
```

Comment:
I believe `string[]` is fine here and elsewhere. `array` would be less precise. From what I understand `string[]` means an array of strings.